system/helpers.php

```diff
@@ -116,3 +116,41 @@ function d($var = false, $to_file = false)
         }
     }
 }
+
+/**
+ * Application container
+ */
+if (! function_exists('app')) {
+    function app()
+    {
+       return App::getContainer();
+    }
+}
+
+/**
+ * Path to root directory
+ *
+ * @return string
+ */
+if (! function_exists('root_path')) {
```

Comment:
We already have the constant [ROOT_PATH](https://github.com/simba77/johncms/blob/develop/system/bootstrap.php#L40), why also a helper?  
He's not needed